src/main.js

```diff
@@ -40,10 +44,14 @@ let load = loader({
   CachePurge: {
     requires: ['cfg', 'monitor'],
     setup: async ({cfg, monitor}) => data.CachePurge.setup({
-      account: cfg.azure.account,
-      table: cfg.app.cachePurgeTableName,
-      credentials: cfg.taskcluster.credentials,
+      tableName: cfg.app.cachePurgeTableName,
       monitor: monitor.prefix('table.purgecaches'),
```

Comment:
I love how we can somehow invent 7 names for a 2-word table, repeatedly across our services..